appveyor.yml

```diff
@@ -36,8 +36,7 @@ artifacts:
 
 environment:
   matrix:
-    - ruby_version: "193"
```

Comment:
Ruby 1.9.3 throws an error and since I don't want to add a specific build for an EOL version for almost 4 years, I've removed it 😬